app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,31 @@
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+<div class="top_login">
+<button><%= link_to "Sign up", new_registration_path(resource_name),
+          class: "sign_up_btn" %></button>
+<h2>Log in</h2>
+<%= f.submit "Log in" %>
+</div>
+<div class="body_login">
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email",
+        class: 'field_input' %>
+  </div>
+
+  <div class="field">
+    <%= f.password_field :password, autocomplete: "current-password", placeholder: "Password",
+        class: 'field_input' %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <% end %>
+
```

Comment:
- [ ] As suggested by the previous reviewer, please improve the styles on the login page in order to match the one in the design given. It looks like what I mentioned on the register page also applies here. That is why I will be sharing the comparison screenshots so that you can easily notice things to change 😃 👍 
**Given design**

![image](https://user-images.githubusercontent.com/90222110/209004934-5f79ba2b-7dca-4a12-8804-1140041c4dcd.png)

**Your Design**
![image](https://user-images.githubusercontent.com/90222110/209004834-a2a0c00a-c09b-4f41-96e9-0f25993d8a18.png)

I hope you noticed that, we don't need the remember me buttons here 😉

Reply:
the changes are made in the deployed version

Reply:
![Screen Shot 2022-12-21 at 15 42 00](https://user-images.githubusercontent.com/99816838/209007702-85fbb39d-bc49-454d-bf97-6dfa0431aa57.png)
![Screen Shot 2022-12-21 at 15 42 59](https://user-images.githubusercontent.com/99816838/209007863-f52aa95c-4fd9-4042-a1c3-e1260e215b77.png)